docs/SDL/OnAppPermissionChanged/index.md

```diff
@@ -9,6 +9,27 @@ Sender
 Purpose
 : Inform HMI that permissions have changed for a specified application.
 
+Notification from SDL to HMI occurs when application permissions were changed. If no permission specified means that application was dissallowed and has to be unregistered.
+
+!!! MUST  
+1.  Request the list of permissions per app for which the consent is required (via `GetListOfPermissions`) when HMI gets the 'user-consent-required' notification per app from `SDL (SDL.OnAppPermissionChanged{appID, appPermissionsConsentNeeded: true})`.  
+2.  Request Policies Manager for the appropriate user-friendly message from Local PT (`SDL.GetUserFriendlyMessage`) to commuticate about permissions changed when HMI gets the list of permissions that require User's consent.
+
+_It is app’s responsibility to inform the user to change the app’s settings if user consent is needed for proper app operation.  
+Any permissions that require user consent are not enabled until a user accepts them. If permissions do not require user consent, they can be applied immediately._ 
+!!!
+
+_**SDL Note**:_   
```

Comment:
Including a newline after this line should make this format better. 

In addition, the `a)`, `b)` format for ordered lists does not show up correctly in markdown. I recommend just using unordered list markers `*` and indenting by 4 spaces in order to format this correctly on the dev portal.